src/skins/categories/aurene-legendary-variants/data.ts

```diff
@@ -0,0 +1,114 @@
+export interface GameDataSkinsCategoriesAureneLegendaryVariants {
+  /** The grouping key for the skins */
+  variant_key: string
```

Comment:
We don't need this one here since we only take a single variant. Let's remove it.